src/scenekit.cs

```diff
@@ -3481,10 +3474,6 @@ interface SCNView : SCNSceneRenderer, SCNTechniqueSupport {
 		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
 		[Export ("rendersContinuously")]
 		bool RendersContinuously { get; set; }
-
-		[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
```

Comment:
This is available in macOS: https://developer.apple.com/documentation/scenekit/scnview/3242930-drawableresizesasynchronously?language=objc